Source/glTFRuntime/Private/glTFRuntimeAssetActor.cpp

```diff
@@ -368,7 +368,7 @@ void AglTFRuntimeAssetActor::Tick(float DeltaTime)
 			FTransform FrameTransform = Pair.Value->GetTransformValue(CurveBasedAnimationsTimeTracker[Pair.Key]);
 			Pair.Key->SetRelativeTransform(FrameTransform);
 		}
-		CurveBasedAnimationsTimeTracker[Pair.Key] += DeltaTime;
+		//CurveBasedAnimationsTimeTracker[Pair.Key] += DeltaTime;
```

Comment:
Can you add a comment to clarify why this is commented out. I suspect it's to force a pause functionality but it may not be immediately clear without the context of this MR.